vita3k/renderer/src/renderer.cpp

```diff
@@ -21,6 +21,8 @@
 
 #include <gxm/functions.h>
 
+#include <util/float_to_half.h>
+
```

Comment:
This include should be on vita3k/renderer/src/vulkan/renderer.cpp not here

Reply:
Indeed, ty